irc/client_lookup_set.go

```diff
@@ -145,7 +145,7 @@ func NewClientDB() *ClientDB {
 	}
 	for _, stmt := range stmts {
 		_, err := db.db.Exec(stmt)
-		if err != nil {
+		if err != nil && !strings.HasSuffix(err.Error(), "already exists") {
```

Comment:
This check for a particular error string (swallowing every other error) is a little dangerous and deserves some comment.